lib/cli/scripts/licenses.ts

```diff
@@ -148,29 +151,34 @@ export default function main(_args: string[], workingDir: string) {
                     console.error(err);
                     reject(err);
                 } else {
+                    const filteredPackages = {};
+
                     // eslint-disable-next-line guard-for-in
                     for (const packageName in packages) {
                         const pack = packages[packageName];
-                        pack['licenseExp'] = pack['licenses']
-                            .toString()
-                            .replace(/\*/g, '')
-                            .replace(/[a-zA-Z0-9\-.]+/g, (match: string) => {
-                                const lowerMatch = match.toLowerCase();
-
-                                if (lowerMatch !== 'and' && lowerMatch !== 'or' && lowerMatch !== 'with') {
-                                    return licenseWithMDLinks(match);
-                                } else {
-                                    return match;
-                                }
-                            });
-
-                        if (!pack['repository']) {
-                            const lastAtSignPos = packageName.lastIndexOf('@');
-                            const mainName = packageName.substring(0, lastAtSignPos);
-
-                            if (missingRepositories[mainName]) {
-                                pack['repository'] = missingRepositories[mainName];
-                            }
+
+                        const lastAtSignPos = packageName.lastIndexOf('@');
```

Comment:
Did you had a chance to fix those changes? And if so how did you test it?

Reply:
Unfortunately I can't support this ticket anymore that's why I asked @MateuszHY to double check with someone who can take care and retest it. Also you can talk with @eromano about running scripts

I changed `node_modules/@alfresco/adf-cli/scripts/licenses.js` then run `scripts/release.sh` and see output.
After I checked that it works correctly (creates new file with 38 dependencies lines) I applied these changes to `lib/cli/scripts/licenses.ts`.
Also you can check with `node dist/libs/cli/scripts/licenses.js` command

Reply:
The functionality was not working as is, @AleksanderSklorz did a great job investigating the issue and we added required fix.